src/components/navigation/z-user-dropdown/index.tsx

```diff
@@ -17,7 +17,7 @@ import { mobileBreakpoint } from "../../../constants/breakpoints";
 })
 export class ZUserDropdown {
   /** logged status flag */
-  @Prop() logged?: boolean;
+  @Prop({mutable: true}) logged?: boolean;
```

Comment:
Why `mutable`?

Reply:
because we need to update the component when this prop change

Reply:
> it's possible to explicitly allow a Prop to be mutated from inside the component, by declaring it as mutable
The prop is never changed from inside the component (as if it was a State variable), so I think the declaration is useless.

Reply:
ok removed